src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,90 @@
+
+![image](energy_use_per_transaction.png)
+
+<p style="text-align: center;"><small><i>Estimate of PoW energy consumption per tx used in figure based on [May 2021 data](https://blog.ethereum.org/2021/05/18/country-power-no-more/), at time of writing the same source suggested up to [175.56 Kwh]((https://digiconomist.net/ethereum-energy-consumption)) </i></small></p>
+
+Let's compare these numbers to a service such as Visa. 100,000 Visa transactions uses 149kWh of energy<sup>[^2]</sup>. Assuming sharding has been implemented, Ethereum's current transaction rate (15 transactions per second) will be increased by at least 64x (the number of shards), not accounting for additional optimization from rollups. A realistic estimate for post-merge, sharded Ethereum with rollups is [25,000 - 100,000](https://twitter.com/VitalikButerin/status/1312905884549300224?s=20) transactions per second. We can use this information to estimate a maximum and minimum energy expenditure per 100,000 transactions.
+
+- 25,000 transactions per second.
+- `100,000 / 25,000 = 4` seconds to process 100,000 transactions.
+
+We can also estimate Ethereum's energy expenditure per second, making a conservative estimate that 10,000 active validators are securing the network (there are over [250,000 validators on the Beacon Chain](https://beaconscan.com/) at the moment, but many validators can operate on a single node. Currently, there are estimated to be 3000-4000 individual nodes, so 10,000 is a conservative estimate for post-merge):
+
+`1.44kWh daily usage * 10,000 network nodes = 14,400kWh` per day.
+There are 86,400 seconds in a day, so `14,400 / 86,400 = 0.1667 kWh` per second.
+
+If we multiply that by the amount of time it takes to process 100,000 transaction: `0.1667 * 4 = 0.667 kWh`.
+
+This is ~0.4% of the energy used by Visa for the same number of transactions, or a reduction in energy expenditure by a factor of ~225.
+
+Repeating the calculation with the maximum transactions-per-second yields 0.1667 kWh per second which is about 0.1% of the energy expenditure of Visa, or a reduction of ~894x.
+
+_It's not entirely accurate to compare based on number of transactions as Ethereum's energy usage is time-based. The energy usage of Ethereum is the same in 1 minute regardless if it does 1 or 1,000 transactions._
+
+_We must also consider that Ethereum isn't limited to simple financial transactions but is also a complete platform built for smart contracts and decentralized applications._
```

Comment:
I feel like we could merge this sentence into the one above, as a set of italicized disclaimers.